src/psbt.cpp

```diff
@@ -30,6 +30,7 @@ bool PartiallySignedTransaction::Merge(const PartiallySignedTransaction& psbt)
         return false;
     }
 
+    assert(*tx_version == psbt.tx_version);
```

Comment:
In a79e519323ee35544e854efd6cbf7ce8577fd4e3 "Implement PSBTv2 field merging": `Assume(...) || return false;`?

Reply:
Done